src/Recommender/Views/TrendingUsersView.jsx

```diff
@@ -135,9 +141,10 @@ function returnProfileForUser(user) {
   }
 
   return null;
-}
+};
 
-if (state.isLoading && !state.hasFetched) {
+if (!state.isLoading && !state.hasLoaded) {
+  returnProfileForUser();
```

Comment:
@scopalaffairs Just curious why we need to call `returnProfileForUser()` here? It doesn't seem to have any affect.

Reply:
You're right, that mistake might have been related to the merge conflict resolving. 
Thanks for catching that!